apim-operator/deploy/controller-configs/controller_conf.yaml

```diff
@@ -89,7 +89,7 @@ data:
   throttleEndpoint: "wso2apim.wso2:32001"
   #Message broker connection URL (related to distributed ratelimiting and token revocation)
   #Format: hostname_of_API_Portal:JMS_port
-  jmsConnectionProvider: "wso2apim.wso2:28230"
+  jmsConnectionProvider: "wso2apim.wso2:32005"
   #Token revocation
```

Comment:
We should not use this port. You can use wso2apim.wso2:5672. From MG, it is possible to connect to this port. For internal service communications we should use internal ports.